src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,125 @@
+package seedu.addressbook.ui;
+
+import seedu.addressbook.commands.CommandResult;
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class Formatter {
+    /**
+     * A decorative prefix added to the beginning of lines printed by AddressBook
+     */
+    public static final String LINE_PREFIX = "|| ";
+    /**
+     * A platform independent line separator.
+     */
+    public static final String LS = System.lineSeparator();
+    public static final String DIVIDER = "===================================================";
+    /**
+     * Format of indexed list item
+     */
+    public static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+    /**
+     * Offset required to convert between 1-indexing and 0-indexing.
+     */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+    /**
+     * Format of a comment input line. Comment lines are silently consumed when reading user input.
+     */
+    public static final String COMMENT_LINE_FORMAT_REGEX = "#.*";
```

Comment:
Let's insert blank lines between a line of code and the header comment for the next line of code so that everything is formatted nicely